particle/particle/particle.py

```diff
@@ -219,6 +218,7 @@ def load_table(cls, filename=None, append=False):
                     P=int(v['P']),
                     C=int(v['C']),
                     anti_flag=int(v['Anti']),
+                    Particle_enum_three_charge=int(v['Charge']),
```

Comment:
How about calling this `_threecharge`? This will make the constructor much nicer and will play nicely with "three_charge" being the public-facing version of this.

Reply:
OK. done.